.github/workflows/go.yml

```diff
@@ -1,39 +1,41 @@
 name: Test diplicity
 
 on:
-    push:
-        branches: [master]
-    pull_request:
-        branches: [master]
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
 
 jobs:
-    build:
-        runs-on: ubuntu-latest
+  build:
+    runs-on: ubuntu-latest
+    container:
+      image: python:2.7.18-buster
```

Comment:
This did not work unfortunately, when I tried it.

Reply:
Oh sorry, I didn't realize you had tried this already.

Reply:
I tried it right before the other CL i mentioned you in. It didn't work because _other_ parts of the cloud tool chain needs Python3 (!).

I got it working (up to a point) when I used `apt get` to install Python2.7, but then I got other errors as mentioned in that PR.

Reply:
So at which point exactly do we need Python2.7? should we try wrapping the start and test scripts in dockerfiles? The Dockerfile I'm using for dev seems to work well.

Reply:
It's very strange. I think most of the gcloud toolchain runs on Python3, but the dev_appserver is something they actually want to get rid of, but so many real customers use it that they can't - but it runs on 2.7 and they don't want to invest into upgrading. Something like that.

Reply:
I'm surprised. If you look at the Dockerfile I added, it doesn't explicitly use Python2.7 at all but it definitely brings up the API and admin apps successfully (was testing it last night).

I can't see why it works in the Dockerfile and wouldn't work in the pipeline.